2019-04-22 09:40:11.091  INFO 5477 --- [main] c.b.fundoo.FundooBackendApplication      : Starting FundooBackendApplication on admin1-desktop with PID 5477 (/home/admin1/Desktop/Tasif/STSWorkspace/BridgeLabz/FundooBackend/target/classes started by admin1 in /home/admin1/Desktop/Tasif/STSWorkspace/BridgeLabz/FundooBackend)
2019-04-22 09:40:11.094  INFO 5477 --- [main] c.b.fundoo.FundooBackendApplication      : No active profile set, falling back to default profiles: default
2019-04-22 09:40:13.285  INFO 5477 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data repositories in DEFAULT mode.
2019-04-22 09:40:13.372  INFO 5477 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 78ms. Found 3 repository interfaces.
2019-04-22 09:40:13.655  INFO 5477 --- [main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.amqp.rabbit.annotation.RabbitBootstrapConfiguration' of type [org.springframework.amqp.rabbit.annotation.RabbitBootstrapConfiguration$$EnhancerBySpringCGLIB$$ec36eb94] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-04-22 09:40:13.729  INFO 5477 --- [main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration$$EnhancerBySpringCGLIB$$25cb2d66] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-04-22 09:40:14.306  INFO 5477 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 8080 (http)
2019-04-22 09:40:14.351  INFO 5477 --- [main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2019-04-22 09:40:14.352  INFO 5477 --- [main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.16]
2019-04-22 09:40:14.357  INFO 5477 --- [main] o.a.catalina.core.AprLifecycleListener   : The APR based Apache Tomcat Native library which allows optimal performance in production environments was not found on the java.library.path: [/usr/java/packages/lib:/usr/lib/x86_64-linux-gnu/jni:/lib/x86_64-linux-gnu:/usr/lib/x86_64-linux-gnu:/usr/lib/jni:/lib:/usr/lib]
2019-04-22 09:40:14.465  INFO 5477 --- [main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2019-04-22 09:40:14.466  INFO 5477 --- [main] o.s.web.context.ContextLoader            : Root WebApplicationContext: initialization completed in 3350 ms
2019-04-22 09:40:14.781  INFO 5477 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
2019-04-22 09:40:14.924  INFO 5477 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
2019-04-22 09:40:15.053  INFO 5477 --- [main] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [
	name: default
	...]
2019-04-22 09:40:15.246  INFO 5477 --- [main] org.hibernate.Version                    : HHH000412: Hibernate Core {5.3.7.Final}
2019-04-22 09:40:15.247  INFO 5477 --- [main] org.hibernate.cfg.Environment            : HHH000206: hibernate.properties not found
2019-04-22 09:40:15.704  INFO 5477 --- [main] o.hibernate.annotations.common.Version   : HCANN000001: Hibernate Commons Annotations {5.0.4.Final}
2019-04-22 09:40:15.906  INFO 5477 --- [main] org.hibernate.dialect.Dialect            : HHH000400: Using dialect: org.hibernate.dialect.MySQL5Dialect
2019-04-22 09:40:17.525  INFO 5477 --- [main] o.h.t.schema.internal.SchemaCreatorImpl  : HHH000476: Executing import script 'org.hibernate.tool.schema.internal.exec.ScriptSourceInputNonExistentImpl@6b71e98f'
2019-04-22 09:40:17.527  INFO 5477 --- [main] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2019-04-22 09:40:18.255  WARN 5477 --- [main] aWebConfiguration$JpaWebMvcConfiguration : spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
2019-04-22 09:40:18.468  INFO 5477 --- [main] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService 'applicationTaskExecutor'
2019-04-22 09:40:18.785  INFO 5477 --- [main] d.s.w.p.DocumentationPluginsBootstrapper : Context refreshed
2019-04-22 09:40:18.808  INFO 5477 --- [main] d.s.w.p.DocumentationPluginsBootstrapper : Found 1 custom documentation plugin(s)
2019-04-22 09:40:18.816  INFO 5477 --- [main] s.d.s.w.s.ApiListingReferenceScanner     : Scanning for api listing references
2019-04-22 09:40:19.019  INFO 5477 --- [main] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [localhost:5672]
2019-04-22 09:40:19.060  INFO 5477 --- [main] o.s.a.r.c.CachingConnectionFactory       : Created new connection: rabbitConnectionFactory#346690d6:0/SimpleConnection@3d0ce151 [delegate=amqp://guest@127.0.0.1:5672/, localPort= 55162]
2019-04-22 09:40:19.142  INFO 5477 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 8080 (http) with context path ''
2019-04-22 09:40:19.146  INFO 5477 --- [main] c.b.fundoo.FundooBackendApplication      : Started FundooBackendApplication in 8.585 seconds (JVM running for 9.76)
2019-04-22 09:41:29.095  INFO 5477 --- [http-nio-8080-exec-1] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring DispatcherServlet 'dispatcherServlet'
2019-04-22 09:41:29.096  INFO 5477 --- [http-nio-8080-exec-1] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
2019-04-22 09:41:29.102  INFO 5477 --- [http-nio-8080-exec-1] o.s.web.servlet.DispatcherServlet        : Completed initialization in 6 ms
2019-04-22 09:42:05.881  INFO 5477 --- [http-nio-8080-exec-6] c.b.f.user.controller.UserController     : userDTO dataUserDto [name=Tasif Mohammed, email=tasifmd96@gmail.com, password=tasifmohammed, mobileNumber=9178184086]
2019-04-22 09:42:05.881  INFO 5477 --- [http-nio-8080-exec-6] c.b.f.u.s.UserServicesImplementation     : UserDto [name=Tasif Mohammed, email=tasifmd96@gmail.com, password=tasifmohammed, mobileNumber=9178184086]
2019-04-22 09:42:05.918  INFO 5477 --- [http-nio-8080-exec-6] o.h.h.i.QueryTranslatorFactoryInitiator  : HHH000397: Using ASTQueryTranslatorFactory
2019-04-22 09:42:06.165  INFO 5477 --- [http-nio-8080-exec-6] c.b.f.u.s.UserServicesImplementation     : User [userId=1, name=Tasif Mohammed, email=tasifmd96@gmail.com, password=$2a$10$G0XuqKzuFwCMCJb9t/Dawu4rwkx0SB/XX38rguq9FKE35gqla1b36, mobileNumber=9178184086, isVarified=false, registeredDate=2019-04-22, updatedDate=null, notes=null, label=null]
2019-04-22 09:42:33.132  INFO 5477 --- [http-nio-8080-exec-7] c.b.f.u.s.UserServicesImplementation     : User : User [userId=1, name=Tasif Mohammed, email=tasifmd96@gmail.com, password=$2a$10$G0XuqKzuFwCMCJb9t/Dawu4rwkx0SB/XX38rguq9FKE35gqla1b36, mobileNumber=9178184086, isVarified=false, registeredDate=2019-04-22, updatedDate=null, notes=[], label=[]]
2019-04-22 09:42:33.132  INFO 5477 --- [http-nio-8080-exec-7] c.b.f.u.s.UserServicesImplementation     : User : User [userId=1, name=Tasif Mohammed, email=tasifmd96@gmail.com, password=$2a$10$G0XuqKzuFwCMCJb9t/Dawu4rwkx0SB/XX38rguq9FKE35gqla1b36, mobileNumber=9178184086, isVarified=true, registeredDate=2019-04-22, updatedDate=2019-04-22, notes=[], label=[]]
2019-04-22 10:05:46.462  INFO 5477 --- [http-nio-8080-exec-1] c.b.f.user.controller.UserController     : User email : tasifmd96@gmail.com
2019-04-22 10:05:46.463  INFO 5477 --- [http-nio-8080-exec-1] c.b.f.u.s.UserServicesImplementation     : Email of user is :tasifmd96@gmail.com
2019-04-22 10:06:25.466  INFO 5477 --- [RMI TCP Connection(29)-127.0.0.1] inMXBeanRegistrar$SpringApplicationAdmin : Application shutdown requested.
2019-04-22 10:06:25.478  INFO 5477 --- [RMI TCP Connection(29)-127.0.0.1] o.s.a.r.l.SimpleMessageListenerContainer : Waiting for workers to finish.
2019-04-22 10:06:25.546  INFO 5477 --- [RMI TCP Connection(29)-127.0.0.1] o.s.a.r.l.SimpleMessageListenerContainer : Successfully waited for workers to finish.
2019-04-22 10:06:25.552  INFO 5477 --- [RMI TCP Connection(29)-127.0.0.1] o.s.a.r.l.SimpleMessageListenerContainer : Shutdown ignored - container is not active already
2019-04-22 10:06:25.552  INFO 5477 --- [RMI TCP Connection(29)-127.0.0.1] o.s.s.concurrent.ThreadPoolTaskExecutor  : Shutting down ExecutorService 'applicationTaskExecutor'
2019-04-22 10:06:25.560  INFO 5477 --- [RMI TCP Connection(29)-127.0.0.1] j.LocalContainerEntityManagerFactoryBean : Closing JPA EntityManagerFactory for persistence unit 'default'
2019-04-22 10:06:25.567  INFO 5477 --- [RMI TCP Connection(29)-127.0.0.1] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown initiated...
2019-04-22 10:06:25.578  INFO 5477 --- [RMI TCP Connection(29)-127.0.0.1] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown completed.
2019-04-22 10:06:28.118  INFO 7835 --- [main] c.b.fundoo.FundooBackendApplication      : Starting FundooBackendApplication on admin1-desktop with PID 7835 (/home/admin1/Desktop/Tasif/STSWorkspace/BridgeLabz/FundooBackend/target/classes started by admin1 in /home/admin1/Desktop/Tasif/STSWorkspace/BridgeLabz/FundooBackend)
2019-04-22 10:06:28.126  INFO 7835 --- [main] c.b.fundoo.FundooBackendApplication      : No active profile set, falling back to default profiles: default
2019-04-22 10:06:28.795  INFO 7835 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data repositories in DEFAULT mode.
2019-04-22 10:06:28.862  INFO 7835 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 58ms. Found 3 repository interfaces.
2019-04-22 10:06:29.055  INFO 7835 --- [main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.amqp.rabbit.annotation.RabbitBootstrapConfiguration' of type [org.springframework.amqp.rabbit.annotation.RabbitBootstrapConfiguration$$EnhancerBySpringCGLIB$$66d98c45] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-04-22 10:06:29.104  INFO 7835 --- [main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration$$EnhancerBySpringCGLIB$$a06dce17] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-04-22 10:06:29.326  INFO 7835 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 8080 (http)
2019-04-22 10:06:29.343  INFO 7835 --- [main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2019-04-22 10:06:29.343  INFO 7835 --- [main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.16]
2019-04-22 10:06:29.348  INFO 7835 --- [main] o.a.catalina.core.AprLifecycleListener   : The APR based Apache Tomcat Native library which allows optimal performance in production environments was not found on the java.library.path: [/usr/java/packages/lib:/usr/lib/x86_64-linux-gnu/jni:/lib/x86_64-linux-gnu:/usr/lib/x86_64-linux-gnu:/usr/lib/jni:/lib:/usr/lib]
2019-04-22 10:06:29.455  INFO 7835 --- [main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2019-04-22 10:06:29.455  INFO 7835 --- [main] o.s.web.context.ContextLoader            : Root WebApplicationContext: initialization completed in 1309 ms
2019-04-22 10:06:29.584  INFO 7835 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
2019-04-22 10:06:29.667  INFO 7835 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
2019-04-22 10:06:29.696  INFO 7835 --- [main] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [
	name: default
	...]
2019-04-22 10:06:29.763  INFO 7835 --- [main] org.hibernate.Version                    : HHH000412: Hibernate Core {5.3.7.Final}
2019-04-22 10:06:29.764  INFO 7835 --- [main] org.hibernate.cfg.Environment            : HHH000206: hibernate.properties not found
2019-04-22 10:06:29.853  INFO 7835 --- [main] o.hibernate.annotations.common.Version   : HCANN000001: Hibernate Commons Annotations {5.0.4.Final}
2019-04-22 10:06:29.923  INFO 7835 --- [main] org.hibernate.dialect.Dialect            : HHH000400: Using dialect: org.hibernate.dialect.MySQL5Dialect
2019-04-22 10:06:30.720  INFO 7835 --- [main] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2019-04-22 10:06:31.291  WARN 7835 --- [main] aWebConfiguration$JpaWebMvcConfiguration : spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
2019-04-22 10:06:31.480  INFO 7835 --- [main] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService 'applicationTaskExecutor'
2019-04-22 10:06:31.675  INFO 7835 --- [main] d.s.w.p.DocumentationPluginsBootstrapper : Context refreshed
2019-04-22 10:06:31.683  INFO 7835 --- [main] d.s.w.p.DocumentationPluginsBootstrapper : Found 1 custom documentation plugin(s)
2019-04-22 10:06:31.690  INFO 7835 --- [main] s.d.s.w.s.ApiListingReferenceScanner     : Scanning for api listing references
2019-04-22 10:06:31.902  INFO 7835 --- [main] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [localhost:5672]
2019-04-22 10:06:31.918  INFO 7835 --- [main] o.s.a.r.c.CachingConnectionFactory       : Created new connection: rabbitConnectionFactory#44ec4a38:0/SimpleConnection@207f7baa [delegate=amqp://guest@127.0.0.1:5672/, localPort= 55720]
2019-04-22 10:06:31.979  INFO 7835 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 8080 (http) with context path ''
2019-04-22 10:06:31.982  INFO 7835 --- [main] c.b.fundoo.FundooBackendApplication      : Started FundooBackendApplication in 4.086 seconds (JVM running for 5.331)
2019-04-22 10:20:43.166  INFO 7835 --- [RMI TCP Connection(17)-127.0.0.1] inMXBeanRegistrar$SpringApplicationAdmin : Application shutdown requested.
2019-04-22 10:20:43.195  INFO 7835 --- [RMI TCP Connection(17)-127.0.0.1] o.s.a.r.l.SimpleMessageListenerContainer : Waiting for workers to finish.
2019-04-22 10:20:44.099  INFO 7835 --- [RMI TCP Connection(17)-127.0.0.1] o.s.a.r.l.SimpleMessageListenerContainer : Successfully waited for workers to finish.
2019-04-22 10:20:44.102  INFO 7835 --- [RMI TCP Connection(17)-127.0.0.1] o.s.a.r.l.SimpleMessageListenerContainer : Shutdown ignored - container is not active already
2019-04-22 10:20:44.102  INFO 7835 --- [RMI TCP Connection(17)-127.0.0.1] o.s.s.concurrent.ThreadPoolTaskExecutor  : Shutting down ExecutorService 'applicationTaskExecutor'
2019-04-22 10:20:44.109  INFO 7835 --- [RMI TCP Connection(17)-127.0.0.1] j.LocalContainerEntityManagerFactoryBean : Closing JPA EntityManagerFactory for persistence unit 'default'
2019-04-22 10:20:44.111  INFO 7835 --- [RMI TCP Connection(17)-127.0.0.1] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown initiated...
2019-04-22 10:20:44.115  INFO 7835 --- [RMI TCP Connection(17)-127.0.0.1] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown completed.
2019-04-22 10:20:45.554  INFO 9022 --- [main] c.b.fundoo.FundooBackendApplication      : Starting FundooBackendApplication on admin1-desktop with PID 9022 (/home/admin1/Desktop/Tasif/STSWorkspace/BridgeLabz/FundooBackend/target/classes started by admin1 in /home/admin1/Desktop/Tasif/STSWorkspace/BridgeLabz/FundooBackend)
2019-04-22 10:20:45.556  INFO 9022 --- [main] c.b.fundoo.FundooBackendApplication      : No active profile set, falling back to default profiles: default
2019-04-22 10:20:46.170  INFO 9022 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data repositories in DEFAULT mode.
2019-04-22 10:20:46.213  INFO 9022 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 40ms. Found 3 repository interfaces.
2019-04-22 10:20:46.388  INFO 9022 --- [main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.amqp.rabbit.annotation.RabbitBootstrapConfiguration' of type [org.springframework.amqp.rabbit.annotation.RabbitBootstrapConfiguration$$EnhancerBySpringCGLIB$$376443e1] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-04-22 10:20:46.437  INFO 9022 --- [main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration$$EnhancerBySpringCGLIB$$70f885b3] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-04-22 10:20:46.636  INFO 9022 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 8080 (http)
2019-04-22 10:20:46.654  INFO 9022 --- [main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2019-04-22 10:20:46.654  INFO 9022 --- [main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.16]
2019-04-22 10:20:46.659  INFO 9022 --- [main] o.a.catalina.core.AprLifecycleListener   : The APR based Apache Tomcat Native library which allows optimal performance in production environments was not found on the java.library.path: [/usr/java/packages/lib:/usr/lib/x86_64-linux-gnu/jni:/lib/x86_64-linux-gnu:/usr/lib/x86_64-linux-gnu:/usr/lib/jni:/lib:/usr/lib]
2019-04-22 10:20:46.735  INFO 9022 --- [main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2019-04-22 10:20:46.735  INFO 9022 --- [main] o.s.web.context.ContextLoader            : Root WebApplicationContext: initialization completed in 1156 ms
2019-04-22 10:20:46.842  INFO 9022 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
2019-04-22 10:20:46.899  INFO 9022 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
2019-04-22 10:20:46.925  INFO 9022 --- [main] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [
	name: default
	...]
2019-04-22 10:20:46.954  INFO 9022 --- [main] org.hibernate.Version                    : HHH000412: Hibernate Core {5.3.7.Final}
2019-04-22 10:20:46.955  INFO 9022 --- [main] org.hibernate.cfg.Environment            : HHH000206: hibernate.properties not found
2019-04-22 10:20:47.052  INFO 9022 --- [main] o.hibernate.annotations.common.Version   : HCANN000001: Hibernate Commons Annotations {5.0.4.Final}
2019-04-22 10:20:47.127  INFO 9022 --- [main] org.hibernate.dialect.Dialect            : HHH000400: Using dialect: org.hibernate.dialect.MySQL5Dialect
2019-04-22 10:20:47.874  INFO 9022 --- [main] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2019-04-22 10:20:48.376  WARN 9022 --- [main] aWebConfiguration$JpaWebMvcConfiguration : spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
2019-04-22 10:20:48.562  INFO 9022 --- [main] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService 'applicationTaskExecutor'
2019-04-22 10:20:48.743  INFO 9022 --- [main] d.s.w.p.DocumentationPluginsBootstrapper : Context refreshed
2019-04-22 10:20:48.751  INFO 9022 --- [main] d.s.w.p.DocumentationPluginsBootstrapper : Found 1 custom documentation plugin(s)
2019-04-22 10:20:48.758  INFO 9022 --- [main] s.d.s.w.s.ApiListingReferenceScanner     : Scanning for api listing references
2019-04-22 10:20:48.921  INFO 9022 --- [main] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [localhost:5672]
2019-04-22 10:20:48.938  INFO 9022 --- [main] o.s.a.r.c.CachingConnectionFactory       : Created new connection: rabbitConnectionFactory#1e749235:0/SimpleConnection@5345552f [delegate=amqp://guest@127.0.0.1:5672/, localPort= 34124]
2019-04-22 10:20:48.981  INFO 9022 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 8080 (http) with context path ''
2019-04-22 10:20:48.984  INFO 9022 --- [main] c.b.fundoo.FundooBackendApplication      : Started FundooBackendApplication in 3.61 seconds (JVM running for 3.99)
2019-04-22 10:21:19.434  INFO 9022 --- [http-nio-8080-exec-1] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring DispatcherServlet 'dispatcherServlet'
2019-04-22 10:21:19.435  INFO 9022 --- [http-nio-8080-exec-1] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
2019-04-22 10:21:19.461  INFO 9022 --- [http-nio-8080-exec-1] o.s.web.servlet.DispatcherServlet        : Completed initialization in 26 ms
2019-04-22 10:21:19.629  INFO 9022 --- [http-nio-8080-exec-1] c.b.f.user.controller.UserController     : LoginDTO data LoginDto [email=tasifmd96@gmail.com, password=tasifmohammed]
2019-04-22 10:21:19.674  INFO 9022 --- [http-nio-8080-exec-1] o.h.h.i.QueryTranslatorFactoryInitiator  : HHH000397: Using ASTQueryTranslatorFactory
2019-04-22 10:21:19.872  INFO 9022 --- [http-nio-8080-exec-1] c.b.f.u.s.UserServicesImplementation     : User Password : $2a$10$G0XuqKzuFwCMCJb9t/Dawu4rwkx0SB/XX38rguq9FKE35gqla1b36
2019-04-22 10:21:57.481  INFO 9022 --- [http-nio-8080-exec-2] c.b.f.u.s.UserServicesImplementation     : Password Reset Successfully
2019-04-22 10:27:56.486  INFO 9022 --- [RMI TCP Connection(11)-127.0.0.1] inMXBeanRegistrar$SpringApplicationAdmin : Application shutdown requested.
2019-04-22 10:27:56.490  INFO 9022 --- [RMI TCP Connection(11)-127.0.0.1] o.s.a.r.l.SimpleMessageListenerContainer : Waiting for workers to finish.
2019-04-22 10:27:57.050  INFO 9022 --- [RMI TCP Connection(11)-127.0.0.1] o.s.a.r.l.SimpleMessageListenerContainer : Successfully waited for workers to finish.
2019-04-22 10:27:57.054  INFO 9022 --- [RMI TCP Connection(11)-127.0.0.1] o.s.a.r.l.SimpleMessageListenerContainer : Shutdown ignored - container is not active already
2019-04-22 10:27:57.055  INFO 9022 --- [RMI TCP Connection(11)-127.0.0.1] o.s.s.concurrent.ThreadPoolTaskExecutor  : Shutting down ExecutorService 'applicationTaskExecutor'
2019-04-22 10:27:57.062  INFO 9022 --- [RMI TCP Connection(11)-127.0.0.1] j.LocalContainerEntityManagerFactoryBean : Closing JPA EntityManagerFactory for persistence unit 'default'
2019-04-22 10:27:57.064  INFO 9022 --- [RMI TCP Connection(11)-127.0.0.1] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown initiated...
2019-04-22 10:27:57.068  INFO 9022 --- [RMI TCP Connection(11)-127.0.0.1] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown completed.
2019-04-22 10:27:58.686  INFO 10119 --- [main] c.b.fundoo.FundooBackendApplication      : Starting FundooBackendApplication on admin1-desktop with PID 10119 (/home/admin1/Desktop/Tasif/STSWorkspace/BridgeLabz/FundooBackend/target/classes started by admin1 in /home/admin1/Desktop/Tasif/STSWorkspace/BridgeLabz/FundooBackend)
2019-04-22 10:27:58.688  INFO 10119 --- [main] c.b.fundoo.FundooBackendApplication      : No active profile set, falling back to default profiles: default
2019-04-22 10:27:59.393  INFO 10119 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data repositories in DEFAULT mode.
2019-04-22 10:27:59.464  INFO 10119 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 62ms. Found 3 repository interfaces.
2019-04-22 10:27:59.700  INFO 10119 --- [main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.amqp.rabbit.annotation.RabbitBootstrapConfiguration' of type [org.springframework.amqp.rabbit.annotation.RabbitBootstrapConfiguration$$EnhancerBySpringCGLIB$$7923cf1f] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-04-22 10:27:59.752  INFO 10119 --- [main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration$$EnhancerBySpringCGLIB$$b2b810f1] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-04-22 10:27:59.961  INFO 10119 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 8080 (http)
2019-04-22 10:27:59.979  INFO 10119 --- [main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2019-04-22 10:27:59.979  INFO 10119 --- [main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.16]
2019-04-22 10:27:59.984  INFO 10119 --- [main] o.a.catalina.core.AprLifecycleListener   : The APR based Apache Tomcat Native library which allows optimal performance in production environments was not found on the java.library.path: [/usr/java/packages/lib:/usr/lib/x86_64-linux-gnu/jni:/lib/x86_64-linux-gnu:/usr/lib/x86_64-linux-gnu:/usr/lib/jni:/lib:/usr/lib]
2019-04-22 10:28:00.102  INFO 10119 --- [main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2019-04-22 10:28:00.103  INFO 10119 --- [main] o.s.web.context.ContextLoader            : Root WebApplicationContext: initialization completed in 1389 ms
2019-04-22 10:28:00.219  INFO 10119 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
2019-04-22 10:28:00.279  INFO 10119 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
2019-04-22 10:28:00.307  INFO 10119 --- [main] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [
	name: default
	...]
2019-04-22 10:28:00.337  INFO 10119 --- [main] org.hibernate.Version                    : HHH000412: Hibernate Core {5.3.7.Final}
2019-04-22 10:28:00.338  INFO 10119 --- [main] org.hibernate.cfg.Environment            : HHH000206: hibernate.properties not found
2019-04-22 10:28:00.426  INFO 10119 --- [main] o.hibernate.annotations.common.Version   : HCANN000001: Hibernate Commons Annotations {5.0.4.Final}
2019-04-22 10:28:00.488  INFO 10119 --- [main] org.hibernate.dialect.Dialect            : HHH000400: Using dialect: org.hibernate.dialect.MySQL5Dialect
2019-04-22 10:28:01.207  INFO 10119 --- [main] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2019-04-22 10:28:01.703  WARN 10119 --- [main] aWebConfiguration$JpaWebMvcConfiguration : spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
2019-04-22 10:28:01.880  INFO 10119 --- [main] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService 'applicationTaskExecutor'
2019-04-22 10:28:02.058  INFO 10119 --- [main] d.s.w.p.DocumentationPluginsBootstrapper : Context refreshed
2019-04-22 10:28:02.067  INFO 10119 --- [main] d.s.w.p.DocumentationPluginsBootstrapper : Found 1 custom documentation plugin(s)
2019-04-22 10:28:02.074  INFO 10119 --- [main] s.d.s.w.s.ApiListingReferenceScanner     : Scanning for api listing references
2019-04-22 10:28:02.257  INFO 10119 --- [main] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [localhost:5672]
2019-04-22 10:28:02.272  INFO 10119 --- [main] o.s.a.r.c.CachingConnectionFactory       : Created new connection: rabbitConnectionFactory#779ef5cb:0/SimpleConnection@5d50e7f6 [delegate=amqp://guest@127.0.0.1:5672/, localPort= 37790]
2019-04-22 10:28:02.316  INFO 10119 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 8080 (http) with context path ''
2019-04-22 10:28:02.319  INFO 10119 --- [main] c.b.fundoo.FundooBackendApplication      : Started FundooBackendApplication in 3.918 seconds (JVM running for 4.325)
2019-04-22 10:36:02.296  INFO 10119 --- [RMI TCP Connection(11)-127.0.0.1] inMXBeanRegistrar$SpringApplicationAdmin : Application shutdown requested.
2019-04-22 10:36:02.305  INFO 10119 --- [RMI TCP Connection(11)-127.0.0.1] o.s.a.r.l.SimpleMessageListenerContainer : Waiting for workers to finish.
2019-04-22 10:36:02.397  INFO 10119 --- [RMI TCP Connection(11)-127.0.0.1] o.s.a.r.l.SimpleMessageListenerContainer : Successfully waited for workers to finish.
2019-04-22 10:36:02.400  INFO 10119 --- [RMI TCP Connection(11)-127.0.0.1] o.s.a.r.l.SimpleMessageListenerContainer : Shutdown ignored - container is not active already
2019-04-22 10:36:02.400  INFO 10119 --- [RMI TCP Connection(11)-127.0.0.1] o.s.s.concurrent.ThreadPoolTaskExecutor  : Shutting down ExecutorService 'applicationTaskExecutor'
2019-04-22 10:36:02.406  INFO 10119 --- [RMI TCP Connection(11)-127.0.0.1] j.LocalContainerEntityManagerFactoryBean : Closing JPA EntityManagerFactory for persistence unit 'default'
2019-04-22 10:36:02.408  INFO 10119 --- [RMI TCP Connection(11)-127.0.0.1] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown initiated...
2019-04-22 10:36:02.412  INFO 10119 --- [RMI TCP Connection(11)-127.0.0.1] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown completed.
2019-04-22 10:36:03.989  INFO 10885 --- [main] c.b.fundoo.FundooBackendApplication      : Starting FundooBackendApplication on admin1-desktop with PID 10885 (/home/admin1/Desktop/Tasif/STSWorkspace/BridgeLabz/FundooBackend/target/classes started by admin1 in /home/admin1/Desktop/Tasif/STSWorkspace/BridgeLabz/FundooBackend)
2019-04-22 10:36:03.991  INFO 10885 --- [main] c.b.fundoo.FundooBackendApplication      : No active profile set, falling back to default profiles: default
2019-04-22 10:36:04.704  INFO 10885 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data repositories in DEFAULT mode.
2019-04-22 10:36:04.751  INFO 10885 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 43ms. Found 3 repository interfaces.
2019-04-22 10:36:04.937  INFO 10885 --- [main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.amqp.rabbit.annotation.RabbitBootstrapConfiguration' of type [org.springframework.amqp.rabbit.annotation.RabbitBootstrapConfiguration$$EnhancerBySpringCGLIB$$41baf2fd] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-04-22 10:36:04.984  INFO 10885 --- [main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration$$EnhancerBySpringCGLIB$$7b4f34cf] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-04-22 10:36:05.198  INFO 10885 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 8080 (http)
2019-04-22 10:36:05.217  INFO 10885 --- [main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2019-04-22 10:36:05.217  INFO 10885 --- [main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.16]
2019-04-22 10:36:05.222  INFO 10885 --- [main] o.a.catalina.core.AprLifecycleListener   : The APR based Apache Tomcat Native library which allows optimal performance in production environments was not found on the java.library.path: [/usr/java/packages/lib:/usr/lib/x86_64-linux-gnu/jni:/lib/x86_64-linux-gnu:/usr/lib/x86_64-linux-gnu:/usr/lib/jni:/lib:/usr/lib]
2019-04-22 10:36:05.306  INFO 10885 --- [main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2019-04-22 10:36:05.307  INFO 10885 --- [main] o.s.web.context.ContextLoader            : Root WebApplicationContext: initialization completed in 1288 ms
2019-04-22 10:36:05.432  INFO 10885 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
2019-04-22 10:36:05.496  INFO 10885 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
2019-04-22 10:36:05.527  INFO 10885 --- [main] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [
	name: default
	...]
2019-04-22 10:36:05.559  INFO 10885 --- [main] org.hibernate.Version                    : HHH000412: Hibernate Core {5.3.7.Final}
2019-04-22 10:36:05.560  INFO 10885 --- [main] org.hibernate.cfg.Environment            : HHH000206: hibernate.properties not found
2019-04-22 10:36:05.666  INFO 10885 --- [main] o.hibernate.annotations.common.Version   : HCANN000001: Hibernate Commons Annotations {5.0.4.Final}
2019-04-22 10:36:05.738  INFO 10885 --- [main] org.hibernate.dialect.Dialect            : HHH000400: Using dialect: org.hibernate.dialect.MySQL5Dialect
2019-04-22 10:36:06.482  INFO 10885 --- [main] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2019-04-22 10:36:06.963  WARN 10885 --- [main] aWebConfiguration$JpaWebMvcConfiguration : spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
2019-04-22 10:36:07.142  INFO 10885 --- [main] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService 'applicationTaskExecutor'
2019-04-22 10:36:07.331  INFO 10885 --- [main] d.s.w.p.DocumentationPluginsBootstrapper : Context refreshed
2019-04-22 10:36:07.340  INFO 10885 --- [main] d.s.w.p.DocumentationPluginsBootstrapper : Found 1 custom documentation plugin(s)
2019-04-22 10:36:07.348  INFO 10885 --- [main] s.d.s.w.s.ApiListingReferenceScanner     : Scanning for api listing references
2019-04-22 10:36:07.537  INFO 10885 --- [main] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [localhost:5672]
2019-04-22 10:36:07.556  INFO 10885 --- [main] o.s.a.r.c.CachingConnectionFactory       : Created new connection: rabbitConnectionFactory#523f3c29:0/SimpleConnection@19b33b0d [delegate=amqp://guest@127.0.0.1:5672/, localPort= 41934]
2019-04-22 10:36:07.642  INFO 10885 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 8080 (http) with context path ''
2019-04-22 10:36:07.645  INFO 10885 --- [main] c.b.fundoo.FundooBackendApplication      : Started FundooBackendApplication in 3.875 seconds (JVM running for 4.34)
2019-04-22 10:37:00.296  INFO 10885 --- [http-nio-8080-exec-1] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring DispatcherServlet 'dispatcherServlet'
2019-04-22 10:37:00.300  INFO 10885 --- [http-nio-8080-exec-1] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
2019-04-22 10:37:00.309  INFO 10885 --- [http-nio-8080-exec-1] o.s.web.servlet.DispatcherServlet        : Completed initialization in 9 ms
2019-04-22 10:37:00.337  INFO 10885 --- [http-nio-8080-exec-1] c.b.f.user.controller.UserController     : User email : tasifmd96@gmail.com
2019-04-22 10:37:00.337  INFO 10885 --- [http-nio-8080-exec-1] c.b.f.u.s.UserServicesImplementation     : Email of user is :tasifmd96@gmail.com
2019-04-22 10:37:00.351  INFO 10885 --- [http-nio-8080-exec-1] o.h.h.i.QueryTranslatorFactoryInitiator  : HHH000397: Using ASTQueryTranslatorFactory
2019-04-22 11:17:25.594  INFO 10885 --- [http-nio-8080-exec-4] c.b.f.u.s.UserServicesImplementation     : Password Reset Successfully
2019-04-22 11:29:39.590  INFO 10885 --- [http-nio-8080-exec-7] c.b.f.user.controller.UserController     : LoginDTO data LoginDto [email=tasifmd96@gmail.com, password=tasifmohammed]
2019-04-22 11:29:39.593  INFO 10885 --- [http-nio-8080-exec-7] c.b.f.u.s.UserServicesImplementation     : User Password : $2a$10$QShSED6Pf5Do7xvnqGfiwurRknV/LKGMfB2VrHXtom3jZGoezstJC
2019-04-22 12:00:39.608  WARN 10885 --- [http-nio-8080-exec-10] .w.s.m.s.DefaultHandlerExceptionResolver : Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.fundoo.response.Response> com.bridgelabz.fundoo.user.controller.UserController.register(com.bridgelabz.fundoo.user.dto.UserDTO) with 4 errors: [Field error in object 'userDTO' on field 'password': rejected value [null]; codes [NotEmpty.userDTO.password,NotEmpty.password,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.password,password]; arguments []; default message [password]]; default message [Please provide password]] [Field error in object 'userDTO' on field 'name': rejected value [null]; codes [NotEmpty.userDTO.name,NotEmpty.name,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.name,name]; arguments []; default message [name]]; default message [please provide your name]] [Field error in object 'userDTO' on field 'mobileNumber': rejected value [1236547]; codes [Pattern.userDTO.mobileNumber,Pattern.mobileNumber,Pattern.java.lang.String,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.mobileNumber,mobileNumber]; arguments []; default message [mobileNumber],[Ljavax.validation.constraints.Pattern$Flag;@28d330bc,org.springframework.validation.beanvalidation.SpringValidatorAdapter$ResolvableAttribute@80b893c]; default message [provide valid mobile number]] [Field error in object 'userDTO' on field 'email': rejected value [edtgtrt]; codes [Email.userDTO.email,Email.email,Email.java.lang.String,Email]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@3d60e502,org.springframework.validation.beanvalidation.SpringValidatorAdapter$ResolvableAttribute@70be1515]; default message [must be a well-formed email address]] ]
2019-04-22 12:01:28.896  WARN 10885 --- [http-nio-8080-exec-1] .w.s.m.s.DefaultHandlerExceptionResolver : Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.fundoo.response.Response> com.bridgelabz.fundoo.user.controller.UserController.register(com.bridgelabz.fundoo.user.dto.UserDTO) with 4 errors: [Field error in object 'userDTO' on field 'mobileNumber': rejected value []; codes [Pattern.userDTO.mobileNumber,Pattern.mobileNumber,Pattern.java.lang.String,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.mobileNumber,mobileNumber]; arguments []; default message [mobileNumber],[Ljavax.validation.constraints.Pattern$Flag;@28d330bc,org.springframework.validation.beanvalidation.SpringValidatorAdapter$ResolvableAttribute@2ad666eb]; default message [provide valid mobile number]] [Field error in object 'userDTO' on field 'email': rejected value [edtgtrt@abccom]; codes [Email.userDTO.email,Email.email,Email.java.lang.String,Email]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@3d60e502,org.springframework.validation.beanvalidation.SpringValidatorAdapter$ResolvableAttribute@6b2b3c7b]; default message [must be a well-formed email address]] [Field error in object 'userDTO' on field 'name': rejected value [null]; codes [NotEmpty.userDTO.name,NotEmpty.name,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.name,name]; arguments []; default message [name]]; default message [please provide your name]] [Field error in object 'userDTO' on field 'password': rejected value [null]; codes [NotEmpty.userDTO.password,NotEmpty.password,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.password,password]; arguments []; default message [password]]; default message [Please provide password]] ]
2019-04-22 12:02:13.378  WARN 10885 --- [http-nio-8080-exec-2] .w.s.m.s.DefaultHandlerExceptionResolver : Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.fundoo.response.Response> com.bridgelabz.fundoo.user.controller.UserController.register(com.bridgelabz.fundoo.user.dto.UserDTO) with 5 errors: [Field error in object 'userDTO' on field 'password': rejected value []; codes [Length.userDTO.password,Length.password,Length.java.lang.String,Length]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.password,password]; arguments []; default message [password],50,6]; default message [password must be at least 6 character and max 50 character]] [Field error in object 'userDTO' on field 'email': rejected value [edtgtrt@abccom]; codes [Email.userDTO.email,Email.email,Email.java.lang.String,Email]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@3d60e502,org.springframework.validation.beanvalidation.SpringValidatorAdapter$ResolvableAttribute@6b9885da]; default message [must be a well-formed email address]] [Field error in object 'userDTO' on field 'password': rejected value []; codes [NotEmpty.userDTO.password,NotEmpty.password,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.password,password]; arguments []; default message [password]]; default message [Please provide password]] [Field error in object 'userDTO' on field 'name': rejected value [null]; codes [NotEmpty.userDTO.name,NotEmpty.name,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.name,name]; arguments []; default message [name]]; default message [please provide your name]] [Field error in object 'userDTO' on field 'mobileNumber': rejected value []; codes [Pattern.userDTO.mobileNumber,Pattern.mobileNumber,Pattern.java.lang.String,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.mobileNumber,mobileNumber]; arguments []; default message [mobileNumber],[Ljavax.validation.constraints.Pattern$Flag;@28d330bc,org.springframework.validation.beanvalidation.SpringValidatorAdapter$ResolvableAttribute@2e9a16cd]; default message [provide valid mobile number]] ]
2019-04-22 12:02:59.722  WARN 10885 --- [http-nio-8080-exec-3] .w.s.m.s.DefaultHandlerExceptionResolver : Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.fundoo.response.Response> com.bridgelabz.fundoo.user.controller.UserController.register(com.bridgelabz.fundoo.user.dto.UserDTO) with 5 errors: [Field error in object 'userDTO' on field 'email': rejected value [edtgtrt@abccom]; codes [Email.userDTO.email,Email.email,Email.java.lang.String,Email]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@3d60e502,org.springframework.validation.beanvalidation.SpringValidatorAdapter$ResolvableAttribute@17efb227]; default message [must be a well-formed email address]] [Field error in object 'userDTO' on field 'password': rejected value []; codes [Length.userDTO.password,Length.password,Length.java.lang.String,Length]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.password,password]; arguments []; default message [password],50,6]; default message [password must be at least 6 character and max 50 character]] [Field error in object 'userDTO' on field 'password': rejected value []; codes [NotEmpty.userDTO.password,NotEmpty.password,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.password,password]; arguments []; default message [password]]; default message [Please provide password]] [Field error in object 'userDTO' on field 'name': rejected value [null]; codes [NotEmpty.userDTO.name,NotEmpty.name,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.name,name]; arguments []; default message [name]]; default message [please provide your name]] [Field error in object 'userDTO' on field 'mobileNumber': rejected value []; codes [Pattern.userDTO.mobileNumber,Pattern.mobileNumber,Pattern.java.lang.String,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.mobileNumber,mobileNumber]; arguments []; default message [mobileNumber],[Ljavax.validation.constraints.Pattern$Flag;@28d330bc,org.springframework.validation.beanvalidation.SpringValidatorAdapter$ResolvableAttribute@23e3971d]; default message [provide valid mobile number]] ]
2019-04-22 12:03:51.617  WARN 10885 --- [http-nio-8080-exec-4] .w.s.m.s.DefaultHandlerExceptionResolver : Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.fundoo.response.Response> com.bridgelabz.fundoo.user.controller.UserController.register(com.bridgelabz.fundoo.user.dto.UserDTO) with 5 errors: [Field error in object 'userDTO' on field 'password': rejected value []; codes [Length.userDTO.password,Length.password,Length.java.lang.String,Length]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.password,password]; arguments []; default message [password],50,6]; default message [password must be at least 6 character and max 50 character]] [Field error in object 'userDTO' on field 'mobileNumber': rejected value []; codes [Pattern.userDTO.mobileNumber,Pattern.mobileNumber,Pattern.java.lang.String,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.mobileNumber,mobileNumber]; arguments []; default message [mobileNumber],[Ljavax.validation.constraints.Pattern$Flag;@28d330bc,org.springframework.validation.beanvalidation.SpringValidatorAdapter$ResolvableAttribute@168dd38a]; default message [provide valid mobile number]] [Field error in object 'userDTO' on field 'email': rejected value [tasif@.com]; codes [Email.userDTO.email,Email.email,Email.java.lang.String,Email]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@3d60e502,org.springframework.validation.beanvalidation.SpringValidatorAdapter$ResolvableAttribute@4c8d8bbc]; default message [must be a well-formed email address]] [Field error in object 'userDTO' on field 'password': rejected value []; codes [NotEmpty.userDTO.password,NotEmpty.password,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.password,password]; arguments []; default message [password]]; default message [Please provide password]] [Field error in object 'userDTO' on field 'name': rejected value [null]; codes [NotEmpty.userDTO.name,NotEmpty.name,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.name,name]; arguments []; default message [name]]; default message [please provide your name]] ]
2019-04-22 12:04:25.102  INFO 10885 --- [RMI TCP Connection(92)-127.0.0.1] inMXBeanRegistrar$SpringApplicationAdmin : Application shutdown requested.
2019-04-22 12:04:25.212  INFO 10885 --- [RMI TCP Connection(92)-127.0.0.1] o.s.a.r.l.SimpleMessageListenerContainer : Waiting for workers to finish.
2019-04-22 12:04:25.549  INFO 10885 --- [RMI TCP Connection(92)-127.0.0.1] o.s.a.r.l.SimpleMessageListenerContainer : Successfully waited for workers to finish.
2019-04-22 12:04:25.580  INFO 10885 --- [RMI TCP Connection(92)-127.0.0.1] o.s.a.r.l.SimpleMessageListenerContainer : Shutdown ignored - container is not active already
2019-04-22 12:04:25.580  INFO 10885 --- [RMI TCP Connection(92)-127.0.0.1] o.s.s.concurrent.ThreadPoolTaskExecutor  : Shutting down ExecutorService 'applicationTaskExecutor'
2019-04-22 12:04:25.591  INFO 10885 --- [RMI TCP Connection(92)-127.0.0.1] j.LocalContainerEntityManagerFactoryBean : Closing JPA EntityManagerFactory for persistence unit 'default'
2019-04-22 12:04:25.620  INFO 10885 --- [RMI TCP Connection(92)-127.0.0.1] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown initiated...
2019-04-22 12:04:25.639  INFO 10885 --- [RMI TCP Connection(92)-127.0.0.1] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown completed.
2019-04-22 12:04:33.033  INFO 16932 --- [main] c.b.fundoo.FundooBackendApplication      : Starting FundooBackendApplication on admin1-desktop with PID 16932 (/home/admin1/Desktop/Tasif/STSWorkspace/BridgeLabz/FundooBackend/target/classes started by admin1 in /home/admin1/Desktop/Tasif/STSWorkspace/BridgeLabz/FundooBackend)
2019-04-22 12:04:33.034  INFO 16932 --- [main] c.b.fundoo.FundooBackendApplication      : No active profile set, falling back to default profiles: default
2019-04-22 12:04:35.730  INFO 16932 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data repositories in DEFAULT mode.
2019-04-22 12:04:35.874  INFO 16932 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 111ms. Found 3 repository interfaces.
2019-04-22 12:04:36.215  INFO 16932 --- [main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.amqp.rabbit.annotation.RabbitBootstrapConfiguration' of type [org.springframework.amqp.rabbit.annotation.RabbitBootstrapConfiguration$$EnhancerBySpringCGLIB$$c1d04b73] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-04-22 12:04:36.268  INFO 16932 --- [main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration$$EnhancerBySpringCGLIB$$fb648d45] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-04-22 12:04:36.938  INFO 16932 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 8080 (http)
2019-04-22 12:04:36.981  INFO 16932 --- [main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2019-04-22 12:04:36.981  INFO 16932 --- [main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.16]
2019-04-22 12:04:37.003  INFO 16932 --- [main] o.a.catalina.core.AprLifecycleListener   : The APR based Apache Tomcat Native library which allows optimal performance in production environments was not found on the java.library.path: [/usr/java/packages/lib:/usr/lib/x86_64-linux-gnu/jni:/lib/x86_64-linux-gnu:/usr/lib/x86_64-linux-gnu:/usr/lib/jni:/lib:/usr/lib]
2019-04-22 12:04:37.203  INFO 16932 --- [main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2019-04-22 12:04:37.204  INFO 16932 --- [main] o.s.web.context.ContextLoader            : Root WebApplicationContext: initialization completed in 4059 ms
2019-04-22 12:04:37.506  INFO 16932 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
2019-04-22 12:04:37.730  INFO 16932 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
2019-04-22 12:04:38.150  INFO 16932 --- [main] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [
	name: default
	...]
2019-04-22 12:04:38.729  INFO 16932 --- [main] org.hibernate.Version                    : HHH000412: Hibernate Core {5.3.7.Final}
2019-04-22 12:04:38.739  INFO 16932 --- [main] org.hibernate.cfg.Environment            : HHH000206: hibernate.properties not found
2019-04-22 12:04:39.912  INFO 16932 --- [main] o.hibernate.annotations.common.Version   : HCANN000001: Hibernate Commons Annotations {5.0.4.Final}
2019-04-22 12:04:40.563  INFO 16932 --- [main] org.hibernate.dialect.Dialect            : HHH000400: Using dialect: org.hibernate.dialect.MySQL5Dialect
2019-04-22 12:04:42.397  INFO 16932 --- [main] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2019-04-22 12:04:43.314  WARN 16932 --- [main] aWebConfiguration$JpaWebMvcConfiguration : spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
2019-04-22 12:04:43.572  INFO 16932 --- [main] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService 'applicationTaskExecutor'
2019-04-22 12:04:43.950  INFO 16932 --- [main] d.s.w.p.DocumentationPluginsBootstrapper : Context refreshed
2019-04-22 12:04:43.966  INFO 16932 --- [main] d.s.w.p.DocumentationPluginsBootstrapper : Found 1 custom documentation plugin(s)
2019-04-22 12:04:43.973  INFO 16932 --- [main] s.d.s.w.s.ApiListingReferenceScanner     : Scanning for api listing references
2019-04-22 12:04:44.192  INFO 16932 --- [main] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [localhost:5672]
2019-04-22 12:04:44.428  INFO 16932 --- [main] o.s.a.r.c.CachingConnectionFactory       : Created new connection: rabbitConnectionFactory#6127ef86:0/SimpleConnection@12e13abd [delegate=amqp://guest@127.0.0.1:5672/, localPort= 57534]
2019-04-22 12:04:44.550  INFO 16932 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 8080 (http) with context path ''
2019-04-22 12:04:44.565  INFO 16932 --- [main] c.b.fundoo.FundooBackendApplication      : Started FundooBackendApplication in 12.601 seconds (JVM running for 17.136)
2019-04-22 12:04:55.306  INFO 16932 --- [http-nio-8080-exec-1] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring DispatcherServlet 'dispatcherServlet'
2019-04-22 12:04:55.309  INFO 16932 --- [http-nio-8080-exec-1] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
2019-04-22 12:04:55.315  INFO 16932 --- [http-nio-8080-exec-1] o.s.web.servlet.DispatcherServlet        : Completed initialization in 6 ms
2019-04-22 12:04:55.448  WARN 16932 --- [http-nio-8080-exec-1] .w.s.m.s.DefaultHandlerExceptionResolver : Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.fundoo.response.Response> com.bridgelabz.fundoo.user.controller.UserController.register(com.bridgelabz.fundoo.user.dto.UserDTO) with 5 errors: [Field error in object 'userDTO' on field 'password': rejected value []; codes [Length.userDTO.password,Length.password,Length.java.lang.String,Length]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.password,password]; arguments []; default message [password],50,6]; default message [password must be at least 6 character and max 50 character]] [Field error in object 'userDTO' on field 'name': rejected value [null]; codes [NotEmpty.userDTO.name,NotEmpty.name,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.name,name]; arguments []; default message [name]]; default message [please provide your name]] [Field error in object 'userDTO' on field 'password': rejected value []; codes [NotEmpty.userDTO.password,NotEmpty.password,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.password,password]; arguments []; default message [password]]; default message [Please provide password]] [Field error in object 'userDTO' on field 'email': rejected value [tasif@.com]; codes [Email.userDTO.email,Email.email,Email.java.lang.String,Email]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@2df68da1,org.springframework.validation.beanvalidation.SpringValidatorAdapter$ResolvableAttribute@1e8e04d2]; default message [must be a well-formed email address]] [Field error in object 'userDTO' on field 'mobileNumber': rejected value []; codes [Pattern.userDTO.mobileNumber,Pattern.mobileNumber,Pattern.java.lang.String,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.mobileNumber,mobileNumber]; arguments []; default message [mobileNumber],[Ljavax.validation.constraints.Pattern$Flag;@275526f7,org.springframework.validation.beanvalidation.SpringValidatorAdapter$ResolvableAttribute@5584bbba]; default message [provide valid mobile number]] ]
2019-04-22 12:05:03.723  WARN 16932 --- [http-nio-8080-exec-2] .w.s.m.s.DefaultHandlerExceptionResolver : Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.fundoo.response.Response> com.bridgelabz.fundoo.user.controller.UserController.register(com.bridgelabz.fundoo.user.dto.UserDTO) with 5 errors: [Field error in object 'userDTO' on field 'mobileNumber': rejected value []; codes [Pattern.userDTO.mobileNumber,Pattern.mobileNumber,Pattern.java.lang.String,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.mobileNumber,mobileNumber]; arguments []; default message [mobileNumber],[Ljavax.validation.constraints.Pattern$Flag;@275526f7,org.springframework.validation.beanvalidation.SpringValidatorAdapter$ResolvableAttribute@3e0c5db9]; default message [provide valid mobile number]] [Field error in object 'userDTO' on field 'email': rejected value [tasif@.com]; codes [Email.userDTO.email,Email.email,Email.java.lang.String,Email]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@2df68da1,org.springframework.validation.beanvalidation.SpringValidatorAdapter$ResolvableAttribute@7b0ec41c]; default message [must be a well-formed email address]] [Field error in object 'userDTO' on field 'password': rejected value []; codes [Length.userDTO.password,Length.password,Length.java.lang.String,Length]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.password,password]; arguments []; default message [password],50,6]; default message [password must be at least 6 character and max 50 character]] [Field error in object 'userDTO' on field 'name': rejected value [null]; codes [NotEmpty.userDTO.name,NotEmpty.name,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.name,name]; arguments []; default message [name]]; default message [please provide your name]] [Field error in object 'userDTO' on field 'password': rejected value []; codes [NotEmpty.userDTO.password,NotEmpty.password,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.password,password]; arguments []; default message [password]]; default message [Please provide password]] ]
2019-04-22 12:05:10.703  WARN 16932 --- [http-nio-8080-exec-3] .w.s.m.s.DefaultHandlerExceptionResolver : Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.fundoo.response.Response> com.bridgelabz.fundoo.user.controller.UserController.register(com.bridgelabz.fundoo.user.dto.UserDTO) with 5 errors: [Field error in object 'userDTO' on field 'mobileNumber': rejected value []; codes [Pattern.userDTO.mobileNumber,Pattern.mobileNumber,Pattern.java.lang.String,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.mobileNumber,mobileNumber]; arguments []; default message [mobileNumber],[Ljavax.validation.constraints.Pattern$Flag;@275526f7,org.springframework.validation.beanvalidation.SpringValidatorAdapter$ResolvableAttribute@1debef9c]; default message [provide valid mobile number]] [Field error in object 'userDTO' on field 'email': rejected value [tasif@.com]; codes [Email.userDTO.email,Email.email,Email.java.lang.String,Email]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@2df68da1,org.springframework.validation.beanvalidation.SpringValidatorAdapter$ResolvableAttribute@58e96245]; default message [must be a well-formed email address]] [Field error in object 'userDTO' on field 'password': rejected value []; codes [Length.userDTO.password,Length.password,Length.java.lang.String,Length]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.password,password]; arguments []; default message [password],50,6]; default message [password must be at least 6 character and max 50 character]] [Field error in object 'userDTO' on field 'name': rejected value [null]; codes [NotEmpty.userDTO.name,NotEmpty.name,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.name,name]; arguments []; default message [name]]; default message [please provide your name]] [Field error in object 'userDTO' on field 'password': rejected value []; codes [NotEmpty.userDTO.password,NotEmpty.password,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.password,password]; arguments []; default message [password]]; default message [Please provide password]] ]
2019-04-22 12:05:11.239  WARN 16932 --- [http-nio-8080-exec-4] .w.s.m.s.DefaultHandlerExceptionResolver : Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.fundoo.response.Response> com.bridgelabz.fundoo.user.controller.UserController.register(com.bridgelabz.fundoo.user.dto.UserDTO) with 5 errors: [Field error in object 'userDTO' on field 'password': rejected value []; codes [Length.userDTO.password,Length.password,Length.java.lang.String,Length]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.password,password]; arguments []; default message [password],50,6]; default message [password must be at least 6 character and max 50 character]] [Field error in object 'userDTO' on field 'email': rejected value [tasif@.com]; codes [Email.userDTO.email,Email.email,Email.java.lang.String,Email]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@2df68da1,org.springframework.validation.beanvalidation.SpringValidatorAdapter$ResolvableAttribute@59f313f3]; default message [must be a well-formed email address]] [Field error in object 'userDTO' on field 'name': rejected value [null]; codes [NotEmpty.userDTO.name,NotEmpty.name,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.name,name]; arguments []; default message [name]]; default message [please provide your name]] [Field error in object 'userDTO' on field 'password': rejected value []; codes [NotEmpty.userDTO.password,NotEmpty.password,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.password,password]; arguments []; default message [password]]; default message [Please provide password]] [Field error in object 'userDTO' on field 'mobileNumber': rejected value []; codes [Pattern.userDTO.mobileNumber,Pattern.mobileNumber,Pattern.java.lang.String,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.mobileNumber,mobileNumber]; arguments []; default message [mobileNumber],[Ljavax.validation.constraints.Pattern$Flag;@275526f7,org.springframework.validation.beanvalidation.SpringValidatorAdapter$ResolvableAttribute@1cf186a9]; default message [provide valid mobile number]] ]
2019-04-22 12:05:11.680  WARN 16932 --- [http-nio-8080-exec-5] .w.s.m.s.DefaultHandlerExceptionResolver : Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.fundoo.response.Response> com.bridgelabz.fundoo.user.controller.UserController.register(com.bridgelabz.fundoo.user.dto.UserDTO) with 5 errors: [Field error in object 'userDTO' on field 'password': rejected value []; codes [Length.userDTO.password,Length.password,Length.java.lang.String,Length]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.password,password]; arguments []; default message [password],50,6]; default message [password must be at least 6 character and max 50 character]] [Field error in object 'userDTO' on field 'name': rejected value [null]; codes [NotEmpty.userDTO.name,NotEmpty.name,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.name,name]; arguments []; default message [name]]; default message [please provide your name]] [Field error in object 'userDTO' on field 'password': rejected value []; codes [NotEmpty.userDTO.password,NotEmpty.password,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.password,password]; arguments []; default message [password]]; default message [Please provide password]] [Field error in object 'userDTO' on field 'mobileNumber': rejected value []; codes [Pattern.userDTO.mobileNumber,Pattern.mobileNumber,Pattern.java.lang.String,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.mobileNumber,mobileNumber]; arguments []; default message [mobileNumber],[Ljavax.validation.constraints.Pattern$Flag;@275526f7,org.springframework.validation.beanvalidation.SpringValidatorAdapter$ResolvableAttribute@4a5e2c68]; default message [provide valid mobile number]] [Field error in object 'userDTO' on field 'email': rejected value [tasif@.com]; codes [Email.userDTO.email,Email.email,Email.java.lang.String,Email]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@2df68da1,org.springframework.validation.beanvalidation.SpringValidatorAdapter$ResolvableAttribute@f5cab47]; default message [must be a well-formed email address]] ]
2019-04-22 12:05:11.890  WARN 16932 --- [http-nio-8080-exec-6] .w.s.m.s.DefaultHandlerExceptionResolver : Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.fundoo.response.Response> com.bridgelabz.fundoo.user.controller.UserController.register(com.bridgelabz.fundoo.user.dto.UserDTO) with 5 errors: [Field error in object 'userDTO' on field 'password': rejected value []; codes [Length.userDTO.password,Length.password,Length.java.lang.String,Length]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.password,password]; arguments []; default message [password],50,6]; default message [password must be at least 6 character and max 50 character]] [Field error in object 'userDTO' on field 'mobileNumber': rejected value []; codes [Pattern.userDTO.mobileNumber,Pattern.mobileNumber,Pattern.java.lang.String,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.mobileNumber,mobileNumber]; arguments []; default message [mobileNumber],[Ljavax.validation.constraints.Pattern$Flag;@275526f7,org.springframework.validation.beanvalidation.SpringValidatorAdapter$ResolvableAttribute@30babf5b]; default message [provide valid mobile number]] [Field error in object 'userDTO' on field 'password': rejected value []; codes [NotEmpty.userDTO.password,NotEmpty.password,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.password,password]; arguments []; default message [password]]; default message [Please provide password]] [Field error in object 'userDTO' on field 'email': rejected value [tasif@.com]; codes [Email.userDTO.email,Email.email,Email.java.lang.String,Email]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@2df68da1,org.springframework.validation.beanvalidation.SpringValidatorAdapter$ResolvableAttribute@75b82728]; default message [must be a well-formed email address]] [Field error in object 'userDTO' on field 'name': rejected value [null]; codes [NotEmpty.userDTO.name,NotEmpty.name,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.name,name]; arguments []; default message [name]]; default message [please provide your name]] ]
2019-04-22 12:05:12.073  WARN 16932 --- [http-nio-8080-exec-7] .w.s.m.s.DefaultHandlerExceptionResolver : Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.fundoo.response.Response> com.bridgelabz.fundoo.user.controller.UserController.register(com.bridgelabz.fundoo.user.dto.UserDTO) with 5 errors: [Field error in object 'userDTO' on field 'mobileNumber': rejected value []; codes [Pattern.userDTO.mobileNumber,Pattern.mobileNumber,Pattern.java.lang.String,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.mobileNumber,mobileNumber]; arguments []; default message [mobileNumber],[Ljavax.validation.constraints.Pattern$Flag;@275526f7,org.springframework.validation.beanvalidation.SpringValidatorAdapter$ResolvableAttribute@3c2a55c9]; default message [provide valid mobile number]] [Field error in object 'userDTO' on field 'password': rejected value []; codes [Length.userDTO.password,Length.password,Length.java.lang.String,Length]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.password,password]; arguments []; default message [password],50,6]; default message [password must be at least 6 character and max 50 character]] [Field error in object 'userDTO' on field 'name': rejected value [null]; codes [NotEmpty.userDTO.name,NotEmpty.name,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.name,name]; arguments []; default message [name]]; default message [please provide your name]] [Field error in object 'userDTO' on field 'email': rejected value [tasif@.com]; codes [Email.userDTO.email,Email.email,Email.java.lang.String,Email]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@2df68da1,org.springframework.validation.beanvalidation.SpringValidatorAdapter$ResolvableAttribute@7928cc28]; default message [must be a well-formed email address]] [Field error in object 'userDTO' on field 'password': rejected value []; codes [NotEmpty.userDTO.password,NotEmpty.password,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.password,password]; arguments []; default message [password]]; default message [Please provide password]] ]
2019-04-22 12:05:12.266  WARN 16932 --- [http-nio-8080-exec-8] .w.s.m.s.DefaultHandlerExceptionResolver : Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.fundoo.response.Response> com.bridgelabz.fundoo.user.controller.UserController.register(com.bridgelabz.fundoo.user.dto.UserDTO) with 5 errors: [Field error in object 'userDTO' on field 'password': rejected value []; codes [Length.userDTO.password,Length.password,Length.java.lang.String,Length]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.password,password]; arguments []; default message [password],50,6]; default message [password must be at least 6 character and max 50 character]] [Field error in object 'userDTO' on field 'name': rejected value [null]; codes [NotEmpty.userDTO.name,NotEmpty.name,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.name,name]; arguments []; default message [name]]; default message [please provide your name]] [Field error in object 'userDTO' on field 'password': rejected value []; codes [NotEmpty.userDTO.password,NotEmpty.password,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.password,password]; arguments []; default message [password]]; default message [Please provide password]] [Field error in object 'userDTO' on field 'mobileNumber': rejected value []; codes [Pattern.userDTO.mobileNumber,Pattern.mobileNumber,Pattern.java.lang.String,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.mobileNumber,mobileNumber]; arguments []; default message [mobileNumber],[Ljavax.validation.constraints.Pattern$Flag;@275526f7,org.springframework.validation.beanvalidation.SpringValidatorAdapter$ResolvableAttribute@2823d43e]; default message [provide valid mobile number]] [Field error in object 'userDTO' on field 'email': rejected value [tasif@.com]; codes [Email.userDTO.email,Email.email,Email.java.lang.String,Email]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@2df68da1,org.springframework.validation.beanvalidation.SpringValidatorAdapter$ResolvableAttribute@6d215f5e]; default message [must be a well-formed email address]] ]
2019-04-22 12:05:12.458  WARN 16932 --- [http-nio-8080-exec-9] .w.s.m.s.DefaultHandlerExceptionResolver : Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.fundoo.response.Response> com.bridgelabz.fundoo.user.controller.UserController.register(com.bridgelabz.fundoo.user.dto.UserDTO) with 5 errors: [Field error in object 'userDTO' on field 'email': rejected value [tasif@.com]; codes [Email.userDTO.email,Email.email,Email.java.lang.String,Email]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@2df68da1,org.springframework.validation.beanvalidation.SpringValidatorAdapter$ResolvableAttribute@7c84992f]; default message [must be a well-formed email address]] [Field error in object 'userDTO' on field 'mobileNumber': rejected value []; codes [Pattern.userDTO.mobileNumber,Pattern.mobileNumber,Pattern.java.lang.String,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.mobileNumber,mobileNumber]; arguments []; default message [mobileNumber],[Ljavax.validation.constraints.Pattern$Flag;@275526f7,org.springframework.validation.beanvalidation.SpringValidatorAdapter$ResolvableAttribute@398618db]; default message [provide valid mobile number]] [Field error in object 'userDTO' on field 'password': rejected value []; codes [Length.userDTO.password,Length.password,Length.java.lang.String,Length]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.password,password]; arguments []; default message [password],50,6]; default message [password must be at least 6 character and max 50 character]] [Field error in object 'userDTO' on field 'name': rejected value [null]; codes [NotEmpty.userDTO.name,NotEmpty.name,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.name,name]; arguments []; default message [name]]; default message [please provide your name]] [Field error in object 'userDTO' on field 'password': rejected value []; codes [NotEmpty.userDTO.password,NotEmpty.password,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.password,password]; arguments []; default message [password]]; default message [Please provide password]] ]
2019-04-22 12:05:25.408  WARN 16932 --- [http-nio-8080-exec-10] .w.s.m.s.DefaultHandlerExceptionResolver : Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.fundoo.response.Response> com.bridgelabz.fundoo.user.controller.UserController.register(com.bridgelabz.fundoo.user.dto.UserDTO) with 5 errors: [Field error in object 'userDTO' on field 'mobileNumber': rejected value []; codes [Pattern.userDTO.mobileNumber,Pattern.mobileNumber,Pattern.java.lang.String,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.mobileNumber,mobileNumber]; arguments []; default message [mobileNumber],[Ljavax.validation.constraints.Pattern$Flag;@275526f7,org.springframework.validation.beanvalidation.SpringValidatorAdapter$ResolvableAttribute@4121138]; default message [provide valid mobile number]] [Field error in object 'userDTO' on field 'password': rejected value []; codes [NotEmpty.userDTO.password,NotEmpty.password,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.password,password]; arguments []; default message [password]]; default message [Please provide password]] [Field error in object 'userDTO' on field 'name': rejected value [null]; codes [NotEmpty.userDTO.name,NotEmpty.name,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.name,name]; arguments []; default message [name]]; default message [please provide your name]] [Field error in object 'userDTO' on field 'email': rejected value [tasif@.com]; codes [Email.userDTO.email,Email.email,Email.java.lang.String,Email]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@2df68da1,org.springframework.validation.beanvalidation.SpringValidatorAdapter$ResolvableAttribute@41108fde]; default message [must be a well-formed email address]] [Field error in object 'userDTO' on field 'password': rejected value []; codes [Length.userDTO.password,Length.password,Length.java.lang.String,Length]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [userDTO.password,password]; arguments []; default message [password],50,6]; default message [password must be at least 6 character and max 50 character]] ]
2019-04-22 12:18:46.671  INFO 16932 --- [http-nio-8080-exec-2] c.b.f.user.controller.UserController     : userDTO dataUserDto [name=Perweez, email=abcde@xyz.com, password=abcdef, mobileNumber=8280946116]
2019-04-22 12:18:46.671  INFO 16932 --- [http-nio-8080-exec-2] c.b.f.u.s.UserServicesImplementation     : UserDto [name=Perweez, email=abcde@xyz.com, password=abcdef, mobileNumber=8280946116]
2019-04-22 12:18:46.714  INFO 16932 --- [http-nio-8080-exec-2] o.h.h.i.QueryTranslatorFactoryInitiator  : HHH000397: Using ASTQueryTranslatorFactory
2019-04-22 12:18:47.113  INFO 16932 --- [http-nio-8080-exec-2] c.b.f.u.s.UserServicesImplementation     : User [userId=2, name=Perweez, email=abcde@xyz.com, password=$2a$10$BTN1DYGxGTqEqhWu798GRuVH92xP/H6AgLCkeXcyREETyD8Qn81Bi, mobileNumber=8280946116, isVarified=false, registeredDate=2019-04-22, updatedDate=null, notes=null, label=null]
2019-04-22 12:28:32.889  WARN 16932 --- [http-nio-8080-exec-5] .w.s.m.s.DefaultHandlerExceptionResolver : Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.bridgelabz.fundoo.response.ResponseToken> com.bridgelabz.fundoo.user.controller.UserController.login(com.bridgelabz.fundoo.user.dto.LoginDTO) throws java.lang.IllegalArgumentException,java.io.UnsupportedEncodingException with 2 errors: [Field error in object 'loginDTO' on field 'password': rejected value []; codes [NotEmpty.loginDTO.password,NotEmpty.password,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [loginDTO.password,password]; arguments []; default message [password]]; default message [Please provide password]] [Field error in object 'loginDTO' on field 'email': rejected value [tas]; codes [Email.loginDTO.email,Email.email,Email.java.lang.String,Email]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [loginDTO.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@2df68da1,org.springframework.validation.beanvalidation.SpringValidatorAdapter$ResolvableAttribute@59fc16f]; default message [must be a well-formed email address]] ]
2019-04-22 12:28:47.348  INFO 16932 --- [http-nio-8080-exec-6] c.b.f.user.controller.UserController     : LoginDTO data LoginDto [email=tasifmd96@gmail.com, password=tasifmohammed]
2019-04-22 12:28:47.397  INFO 16932 --- [http-nio-8080-exec-6] c.b.f.u.s.UserServicesImplementation     : User Password : $2a$10$QShSED6Pf5Do7xvnqGfiwurRknV/LKGMfB2VrHXtom3jZGoezstJC
